# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/HttpBackgroundChannelParent.h
# Commit: 673030046319
# Full Hash: 673030046319948020e1e2100e2db3cf753dbaba
# Author: Junior Hsu <juhsu@mozilla.com>
# Date: 2020-06-30 10:00:42
# Regressor Bug: 1633935
# File Overlap Count: 1
# Description:
#   Bug 1633935 - P9 send OnAfterLastPart via pBg IPC to avoid race between OS*R, r=mayhemer,necko-reviewers
#   
#   Depends on D77750
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D77751
# ==============================================================================

diff -r 045271cd2ca6 -r 673030046319 netwerk/protocol/http/HttpBackgroundChannelParent.h
--- a/netwerk/protocol/http/HttpBackgroundChannelParent.h	Tue Jun 30 00:33:41 2020 +0000
+++ b/netwerk/protocol/http/HttpBackgroundChannelParent.h	Tue Jun 30 00:33:54 2020 +0000
@@ -57,6 +57,9 @@
                      const nsHttpHeaderArray& aResponseTrailers,
                      const nsTArray<ConsoleReportCollected>& aConsoleReports);
 
+  // To send OnAfterLastPart message over background channel.
+  bool OnAfterLastPart(const nsresult aStatus);
+
   // To send OnProgress message over background channel.
   bool OnProgress(const int64_t aProgress, const int64_t aProgressMax);
 